godot/Cargo.toml

```diff
@@ -18,6 +18,8 @@ formatted = ["godot-core/codegen-fmt"]
 lazy-function-tables = ["godot-core/codegen-lazy-fptrs"]
 serde = ["godot-core/serde"]
 
+docs = ["godot-macros/docs"]
```

Comment:
i think this could have a better name, like `generate-user-docs` maybe? as-is `docs` may imply a lot of things which it doesn't do, for instance enabling godot-docs in rust.